connectivity/tests/world.go

```diff
@@ -10,14 +10,19 @@ import (
 	"github.com/cilium/cilium-cli/connectivity/check"
 )
 
-// PodToWorld sends multiple HTTP(S) requests to one.one.one.one
-// from random client Pods.
-func PodToWorld() check.Scenario {
-	return &podToWorld{}
+// PodToWorld sends multiple HTTP(S) requests to ExternalTarget default one.one.one.one
```

Comment:
Let's omit the default:

```suggestion
// PodToWorld sends multiple HTTP(S) requests to ExternalTarget
```